routes/api/items.js

```diff
@@ -0,0 +1,42 @@
+const express = require( 'express' )
+const router = express.Router()
+
+const { allItemsQuery, filteredItemsQuery, respondWithItemsApi } = require( '../items/item_response' )
+const { buildTree } = require( '../items/tree_creation' )
+
+router.get( '/list/:user_id/:email', ( request, response ) => {
+  const Item = request.app.get( 'models' ).Item
+
+  const { params, query } = request
+
+  Item.findAll( allItemsQuery( params.user_id ))
```

Comment:
There's a lot of duplication of code between these endpoints, and the existing endpoints.  We should avoid code duplication...